### ISSUE TITLE
Re-training: Initial first learning rate of learning rate schedule is taken 

### ISSUE THREAD

**Issue by AnFrBo** [DeepLabCut/DeepLabCut#1520]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hey @jeylau,

I think the issue is in between a bug and a feature request. When retraining, I noticed that the first learning rate of the learning rate schedule is taken.

For instance: I trained a model to 60k iterations with the default learning rate schedule:
- - 0.0001
  - 7500
- - 5.0e-05
  - 12000
- - 1.0e-05
  - 200000

When I trained the model with additional iterations, taken the already trained weights of 60k iterations, it took automatically the first learning rate of 0.0001. Since 7500 iterations were not reached (as it was already at 60k iterations), the learning rate did not change to 5.0e-05 nor 1.0e-05. I attached an image where you can see the influence on the loss.

![model_loss](https://user-images.githubusercontent.com/39856231/134776735-9fa050f8-24aa-48b5-91f3-3ca295e3f60c.png)

I am sure this was not implemented intentionally. As a solution, maybe you can write it down somewhere that the learning rate has to be adjusted manually in the pose_conf.yaml file or you could add a if statement or an extra parameter to the function? What do you think?

### Operating System

Ubuntu 18.04. LTS

### DeepLabCut version

2.2.0.2

### DeepLabCut mode

multi animal

### Device type

NVIDIA GeForce RTX 3090

### Steps To Reproduce

Run iteration > 7500, stop, add saved weights of last iterations as init_weights. Start training again. Look at learning rate. 

### Relevant log output

```shell
iteration: 60500	 loss: 0.0016	 scmap loss: 0.0014	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 61000	 loss: 0.0015	 scmap loss: 0.0014	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 61500	 loss: 0.0015	 scmap loss: 0.0014	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 62000	 loss: 0.0015	 scmap loss: 0.0014	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 62500	 loss: 0.0015	 scmap loss: 0.0014	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 63000	 loss: 0.0014	 scmap loss: 0.0013	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 63500	 loss: 0.0014	 scmap loss: 0.0013	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
iteration: 64000	 loss: 0.0015	 scmap loss: 0.0013	 locref loss: 0.0000	 limb loss: 0.0001	 lr: 0.0001
(...)
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
HI @AnFrBo that is actually by design so not to worry You can see here, in Figure 5 (I would recommend checking out this paper, as you use EfficientNets, this is critical): https://openaccess.thecvf.com/content/WACV2021/papers/Mathis_Pretraining_Boosts_Out-of-Domain_Robustness_for_Pose_Estimation_WACV_2021_paper.pdf
